nanover-rs/src/tracked_simulation/openmm.rs

```diff
@@ -161,12 +161,21 @@ impl TrackedSimulation for TrackedOpenMMSimulation {
     ) -> Result<(), BroadcastSendError> {
         let mut frame = self.simulation.to_framedata(with_velocities, with_forces);
         let energy_total = self.user_energies();
+        let potential_energy = self.simulation.get_potential_energy();
+        let potential_energy_correction =
+            compute_potential_energy_correction(self.user_forces(), &self.simulation);
         frame
             .insert_number_value("energy.user.total", energy_total)
             .unwrap();
         frame
             .insert_number_value("system.reset.counter", self.reset_counter as f64)
             .unwrap();
+        frame
+            .insert_number_value(
+                "energy.potential_corrected",
```

Comment:
Might want to make it `energy.potential.corrected` since we use dots everywhere else and not underscores? Doesn't matter too much, especially if this is to be a temporary field for the meantime, so take or leave this suggestion.